Fortran missing again on Mac CI nodes?

Fortran missing again on Mac CI nodes?

See this build log from #966, failing make info
https://github.com/madgraph5/madgraph4gpu/actions/runs/10385768660/job/28755634712?pr=966
```
Run make BACKEND=cppnone OMPFLAGS= FPTYPE=d -C epochX/cudacpp/gg_ttgg.mad/SubProcesses/P1_gg_ttxgg -f cudacpp.mk info
/bin/bash: gfortran-11: command not found
BACKEND='cppnone'
OMPFLAGS=
FPTYPE='d'
HELINL='0'
HRDCOD='0'
HASCURAND=hasNoCurand
HASHIPRAND=hasNoHiprand
Building in BUILDDIR=. for tag=none_d_inl0_hrd0_hasNoCurand_hasNoHiprand (USEBUILDDIR != 1)

Darwin Mac-1723630258175.local arm
machdep.cpu.brand_string: Apple M1 (Virtual)
hw.physicalcpu: 3
hw.logicalcpu: 3

USECCACHE=

GPUCC=

CXX=c++
c++ --version
Apple clang version 15.0.0 (clang-1500.3.9.4)
Target: arm64-apple-darwin23.6.0
Thread model: posix
InstalledDir: /Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

FC=gfortran-11
gfortran-11 --version
make: gfortran-11: No such file or directory
make: *** [info] Error 1
Error: Process completed with exit code 2.
```

The nodes are not all configured in the same way, because this one succeeds make info
https://github.com/madgraph5/madgraph4gpu/actions/runs/10385768660/job/28755635686?pr=966
```
Run make BACKEND=cppsse4 OMPFLAGS= FPTYPE=d -C epochX/cudacpp/gg_ttgg.mad/SubProcesses/P1_gg_ttxgg -f cudacpp.mk info
BACKEND='cppsse4'
OMPFLAGS=
FPTYPE='d'
HELINL='0'
HRDCOD='0'
HASCURAND=hasNoCurand
HASHIPRAND=hasNoHiprand
Building in BUILDDIR=. for tag=sse4_d_inl0_hrd0_hasNoCurand_hasNoHiprand (USEBUILDDIR != 1)

Darwin Mac-1723630392902.local arm
machdep.cpu.brand_string: Apple M1 (Virtual)
hw.physicalcpu: 3
hw.logicalcpu: 3

USECCACHE=

GPUCC=

CXX=c++
c++ --version
Apple clang version 15.0.0 (clang-1500.3.9.4)
Target: arm64-apple-darwin23.6.0
Thread model: posix
InstalledDir: /Applications/Xcode_15.4.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin

FC=gfortran-11
gfortran-11 --version
GNU Fortran (Homebrew GCC 11.4.0) 11.4.0
Copyright (C) 2021 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```

Well, thanks Mac...

--- a/.github/workflows/c-cpp.yml
+++ b/.github/workflows/c-cpp.yml
@@ -38,7 +38,7 @@ jobs:
   CPU_MAC:
     runs-on: macos-latest
     env:
-      FC: gfortran-11
+      FC: gfortran-14 # see #971
     strategy:
       matrix:
         folder: [ epochX/cudacpp/ee_mumu.mad/SubProcesses/P1_epem_mupmum, epochX/cudacpp/gg_ttgg.mad/SubProcesses/P1_gg_ttxgg ]

